src/Misc/Part.cpp

```diff
@@ -1038,6 +1042,7 @@ int Part::saveXML(const char *filename)
     xml.endbranch();
 
     int result = xml.saveXMLfile(filename, gzip_compression);
+    strcpy(loaded_file,filename);
```

Comment:
Here, and also in `loadXMLinstrument`, I'd rather user `strncpy` for safety. If you change it, please make sure it's null terminated: `Warning: If there is no null byte among the first n bytes of src, the string placed in dest will not be null-terminated. (man page)`

Reply:
question is where does the n come from. 
in case of loading the filename comes from osc:

Middleware.spp:1432
const char *file  = rtosc_argument(msg,1).s;

Reply:
The size of the source string is not important, the size of the destination string (`sizeof(loaded_file)`) is, since we don't want to overwrite the destination string.

Reply:
ok